### DISCUSSION TITLE
This is a Q&A

### DISCUSSION THREAD

**Discussion by willhiorns** [UKHomeOffice-test/docs-as-code-nunjucks#28] [Q&A] [answered]
Just seeing what fields and capabilities are surfaced.

**Comment by willhiorns** [comment]
Well, this is an answer.  A really good one.  It's the first one I created.  You should vote for this.

**Comment by willhiorns** [comment]
This is quite a poor answer.  Turns out I was talking nonsense.   It's the second one I created.  

Don't vote for it.  It will delete the Internet.

**Comment by willhiorns** [comment]
Don't vote for this one either, but give it lots of thumbs ups.  I wonder what will happen then.

It's the third one I created.

**Reply by willhiorns** [reply]
@robertdeniszczyc2 would you please 👍 this one?

**Reply by robertdeniszczyc2** [reply]
Done

**Reply by willhiorns** [reply]
This answer doesn't sort to Top despite having the most 👍 , because it has no proper up-votes (up-arrows).

Seems to me that thumbs are sentiment markers rather than votes.  I wonder if there's any role-based difference in ability to toggle them?

**Comment by willhiorns** [comment, marked as answer]
This is a brilliant answer, even better than the first one (what an idiot he was, I've just thumbs-downed that nonsense).   It's the fourth one I created.  Please vote for it even more than the first.

I can't work out how to promote it to the head of the list.  Upvotes and thumbs seem not to affect the "Top" view.

Might have to consult the docs.

**Reply by RobJohnsonHO** [reply]
Top seems to work for me once the numbers line up.  the 3 up voted answers are now on top for me!

**Reply by willhiorns** [reply]
Can you tell me whether you, as a peer on this repo, can see the "Mark as answer" button?  Or is it only visible to the author at the moment?

**Reply by robertdeniszczyc2** [reply]
I can see "Mark as answer"

**Reply by RobJohnsonHO** [reply]
And me

**Reply by willhiorns** [reply]
> Top seems to work for me once the numbers line up. the 3 up voted answers are now on top for me!

Some "Discussion" on this feature here: https://github.com/github/feedback/discussions/3328 - seems like it might only work for more recent timeframes at the moment.  But yes, the up-arrow votes seem to be driving it.

**Reply by robertdeniszczyc2** [reply]
Also now this has more votes than the first answer the "Top" view put this at the top of the list

**Reply by willhiorns** [reply]
Thread here of people talking about difference of thumbs vs up-arrow voting.  I want to see if these behave differently after answering the question - https://github.com/github/feedback/discussions/3474?sort=top

**Reply by willhiorns** [reply]
> I can see "Mark as answer"

Thanks - that's great to know.

**Reply by willhiorns** [reply]
I'm going to accept this answer, and then we can see what happens to the functionality on this discussion.

**Reply by willhiorns** [reply]
OK, there's now an "Unmark as answer".

Thumbs and up-votes all still behave same.

The visibility of "trending" answers in Top (with recent filter) is interesting.  Does that give the a lens through which we can see new, better answers that outweigh an old one which loads of people voted for in the past?  Suspect so.
